app/controllers/organisers_controller.rb

```diff
@@ -2,12 +2,58 @@ class OrganisersController < ApplicationController
    before_action :require_organiser
 
   def index
-    @organisers = User.where(role: 1)
+    display_organisers
+  end
+  
+  def create
+    @user = User.find_by(email: params[:email])
```

Comment:
why are you using `find_by` if we're creating new user here so obviously it won't exist in database yet?